### PR TITLE
cargo-deb: init at 1.21.1

### PR DESCRIPTION
**Pull request by Br1ght0ne**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/73230)

### CODE REVIEW

**Review by JohnAZoidberg — CHANGES_REQUESTED** [review]

File: pkgs/tools/package-management/cargo-deb/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, runCommand, fetchFromGitHub, rustPlatform, Security }:
```
[the remark this answers is not in the document]
Comment (filalex77) [reply pkgs/tools/package-management/cargo-deb/default.nix]: That slipped from me. Thanks!

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM
executable seems to work

```
[4 built, 0.0 MiB DL]
https://github.com/NixOS/nixpkgs/pull/73230
1 package were build:
cargo-deb
```

**Comment by jonringer** [comment]
can you squash the commits?
```
git reset HEAD^
git add pkgs/
git commit --amend --no-edit
git push --force <fork> <branch>
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build cargo-deb

**Pull request merged**